docs/reference/supported-environments.md

```diff
@@ -33,52 +33,108 @@ For example, 1.2+ means support for the minor version 2, and any higher minors (
 
 ## Camunda 8 Self-Managed
 
-We highly recommend running Camunda 8 Self-Managed in a Kubernetes environment. We provide officially supported [Helm Charts](/self-managed/platform-deployment/helm-kubernetes/overview.md) for this. Please follow the [Installation Guide](/self-managed/platform-deployment/overview.md) to learn more about installation possibilities.
+### Minimum requirements
+
+The following are the minimum supported requirements for cluster specification and dependencies:
+
+- 4vCPUs, 15 GB Memory
```

Comment:
small note. vCPU will not be applicable on bare-metal installs. maybe we can call our "4 (v)CPU"

Reply:
Sure! @theburi Do we have documentation on bare-metal installs? Would this be [manual installation?](https://docs.camunda.io/docs/self-managed/platform-deployment/manual/)

Reply:
Generally, we would expect users to use Manual installation, but they could also use Docker-compose (Docker standalone). Looking at my comment today, CPU to vCPU relationship is hard to describe as there are different calculations. So I think it becomes too nuanced and we should not go there.